kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/convert/BigQuerySchemaConverter.java

```diff
@@ -120,33 +119,37 @@ public BigQuerySchemaConverter(boolean allFieldsNullable) {
     return com.google.cloud.bigquery.Schema.of(fields);
   }
 
-  private void throwOnCycle(Schema kafkaConnectSchema, List<Schema> seenSoFar) {
+  private boolean throwOnCycle(Schema kafkaConnectSchema, List<Schema> seenSoFar) {
```

Comment:
Nit: if we're not throwing an exception from this method anymore and are instead returning a boolean indicating whether the schema has a cycle or not, the method should be renamed to something like `schemaContainsCycle`.

Reply:
Faire point. Done